crates/telio-task/src/task/runtime.rs

```diff
@@ -200,25 +139,25 @@ where
 
     /// Stop task
     pub async fn stop(mut self) -> StopResult<S::Err> {
-        println!("task stopped - {}", S::NAME);
-        self.stop.notify_one();
+        // We can ignore error, if fail to send stop
+        // it means that inner task was already stopped
+        // and reciever part was dropped
+        let _ = self.stop.send(StopKind::Manual).await;
```

Comment:
maybe a warning here would be useful to see if we have any such cases where we try to explicitly stop tasks no longer running?